spyderlib/widgets/editor.py

```diff
@@ -451,14 +334,15 @@ def __init__(self, parent, actions):
 
         self.data = []
 
-        filelist_action = create_action(self, _("File list management"),
-                                 icon=get_icon('filelist.png'),
-                                 triggered=self.open_filelistdialog)
+        fileswitcher_action = create_action(self, _("File switcher..."),
+                        icon=get_icon('filelist.png'),
+                        triggered=self.open_fileswitcher_dlg)

```

Comment:
Indentation is wrong here, it should be aligned after the first `(`.
See https://www.python.org/dev/peps/pep-0008/#indentation